runtime/src/main/cpp/Memory.h

```diff
@@ -83,10 +85,18 @@ struct ContainerHeader {
     refCount_ = (refCount_ & ~CONTAINER_TAG_MASK) | CONTAINER_TAG_FROZEN;
   }
 
+  inline void makeShareable() {
+      refCount_ = (refCount_ & ~CONTAINER_TAG_MASK) | CONTAINER_TAG_ATOMIC;
+  }
+
   inline bool permanentOrFrozen() const {
```

Comment:
Is it needed? For example, is it ok to use it here?
https://github.com/JetBrains/kotlin-native/blob/dfbafd420bafdfe3954da56609b29efbdca8f81f/runtime/src/main/cpp/Memory.cpp#L1817

Reply:
I left that code intentionally, as not sure that with current GC we can use shared objects in Konan_ensureAcyclicAndSet (and also it is useless, as cycles may appear concurrently after the check).